CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+### Changed
+* **BREAKING: core + sources:** Update Documentheader ([#1515](https://github.com/dso-toolkit/dso-toolkit/issues/1515))
```

Comment:
Graag nog een samenvatting van wat er BREAKING is in het PR

Reply:
Done